java-components/build-request-processor/src/main/resources/gradle/repositories.gradle

```diff
@@ -3,17 +3,20 @@ apply plugin: RepositoryPlugin
 class RepositoryPlugin implements Plugin<Gradle> {
 
     private static String ENTERPRISE_REPOSITORY_URL = System.getenv("CACHE_URL") != null ? System.getenv("CACHE_URL") : "http://localhost:8080/v2/cache/rebuild-default,gradle,gradleplugins/0"
-    private static String BINTRAY_URL = ENTERPRISE_REPOSITORY_URL + "/hacbs-bintray"
 
     void apply(Gradle gradle) {
         def fixRepositories = {
             def bintray = false
             all { ArtifactRepository repo ->
                 if (!(repo instanceof MavenArtifactRepository) ||
-                        (repo.url.toString() != ENTERPRISE_REPOSITORY_URL && repo.url.toString() != BINTRAY_URL )) {
+                        (repo.url.toString() != ENTERPRISE_REPOSITORY_URL)) {
                     if (repo.toString().contains("PluginArtifactRepository") || (Boolean.getBoolean("ALLOW_IVY_REPO") && repo.toString().contains("DefaultIvyArtifactRepository"))) {
                         return
                     }
+                    if (repo instanceof IvyArtifactRepository) {
```

Comment:
What's up with this change? Is it related?

Reply:
Oops, this was accidentally commited.